src/Gaufrette/Stream/Local.php

```diff
@@ -12,9 +12,9 @@
  */
 class Local implements Stream
 {
-    private $path;
-    private $mode;
-    private $fileHandle;
+    protected $path;
+    protected $mode;
+    protected $fileHandle;
```

Comment:
Making things protected means they become extension points which need to be taken into account for backward compatibility. And extension points by inheritance are the worse ones regarding BC layers (we implement BC layers in Symfony since years, and the ones involving inheritance and BC for protected things have always been the worse ones to implement)